nifi-nar-bundles/nifi-standard-services/nifi-record-serialization-services-bundle/nifi-record-serialization-services/src/test/resources/text/test_lines.txt

```diff
@@ -0,0 +1,7 @@
+Header line 1
```

Comment:
Would it make sense to test multiple sequential empty lines instead of just a single?